lzero/worker/muzero_evaluator.py

```diff
@@ -324,14 +324,32 @@ def eval(
                     for env_id, t in timesteps.items():
                         obs, reward, done, info = t.obs, t.reward, t.done, t.info
 
+                        eps_steps_lst[env_id] += 1
+
+                        if hasattr(self._policy.get_attribute('collect_model'), 'world_model'):
+                            if hasattr(self.policy_config,
+                                       'sample_type') and self.policy_config.sample_type == 'episode':
+                                clear_interval = 2000
+                            else:
+                                clear_interval = 200
+                            # print(f'clear_interval: {clear_interval}')
+                            if eps_steps_lst[env_id] % clear_interval == 0:
+                                print(f'clear_interval: {clear_interval}')
```

Comment:
call `self._policy.reset()` to implement this parts of code, i.e. `_reset_collect` method of policy

Reply:
done